src/clj/ataru/hakija/application_email_confirmation.clj

```diff
@@ -1,24 +1,41 @@
 (ns ataru.hakija.application-email-confirmation
   "Application-specific email confirmation init logic"
   (:require
-   [taoensso.timbre :as log]
-   [selmer.parser :as selmer]
-   [ataru.applications.application-store :as application-store]
-   [ataru.background-job.job :as job]
-   [ataru.hakija.background-jobs.hakija-jobs :as hakija-jobs]
-   [ataru.hakija.background-jobs.email-job :as email-job]))
+    [taoensso.timbre :as log]
+    [selmer.parser :as selmer]
+    [ataru.applications.application-store :as application-store]
+    [ataru.background-job.job :as job]
+    [ataru.hakija.background-jobs.hakija-jobs :as hakija-jobs]
+    [ataru.hakija.background-jobs.email-job :as email-job]
+    [oph.soresu.common.config :refer [config]]
+    [clojure.java.io :as io]
+    [clojure.edn :as edn])
+  (:import [java.io FileReader PushbackReader]))
 
-(def ^:private subject
-  {:fi "Opintopolku.fi - Hakemuksesi on vastaanotettu"
-   :sv "Opintopolku.fi - Din ansökan har tagits emot"
-   :en "Opintopolku.fi - Your application has been received"})
+(defn- get-translations [lang]
+  (with-open [reader (-> "translations/email_confirmation.edn"
+                         io/resource
+                         io/file
+                         FileReader.
+                         PushbackReader.)]
```

Comment:
Btw. PushbackReader is for cases, where you actually want to "undead" a byte from the stream, e.g. when implementing compilers.